liblouis/internal.h

```diff
@@ -72,7 +72,8 @@ typedef struct intCharTupple {
 #define MAX_EMPH_CLASSES 10  // {emph_1...emph_10} in typeforms enum (liblouis.h)
 
 typedef unsigned int TranslationTableOffset;
-#define OFFSETSIZE sizeof(TranslationTableOffset)
+typedef unsigned long long TranslationTableData;
```

Comment:
It would be nice to have a comment explaining why we have this type. It's used only in one place so can be quite confusing.

Reply:
Right, I added a comment